ztp/kube-compare-reference/optional/storage/StorageLV.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: "local.storage.openshift.io/v1"
+kind: "LocalVolume"
+metadata:
+  name: "local-disks"
+  namespace: "openshift-local-storage"
+spec:
+  logLevel: Normal
+  managementState: Managed
+  storageClassDevices:
+    {{- range .spec.storageClassDevices }}
+    - storageClassName: {{ .storageClassName }}
+      volumeMode: Filesystem
+      fsType: xfs
+      devicePaths:
+{{ .devicePaths | toYaml | indent 8 }}
```

Comment:
Ideally we would validate that they use /dev/disk/by-path for disk identification. Other references are not stable.

Reply:
Checking for `/dev/disk/by-path`